### PR TITLE
Add GraphQL mutation example

### PR DESCRIPTION
**Pull request by tandrup**

### CODE REVIEW

File: guides/sdk-docs/public-apis/graphql-api.md [hunk 1]
```diff
@@ -43,10 +45,10 @@ export default config;
...
-### 4. Create Graphql Query 
-Now you are ready to create React hooks from your GraphQL queries, just copy your query or the below query to your src folder in the libs/[feature-name]/[name-of-your-extension]/src and name it demo.graphql
+## 4. Create Graphql Query 
+Now you are ready to create React hooks from your GraphQL queries, just copy your query or the below query to your src folder in the `libs/[feature-name]/[name-of-your-extension]/src` and name it `demo.graphql`.
 
-```Text Graphql
+```graphql
 query GetAssetsByProductionYear($first: Int, $productionYears: [String!]) {
   assets(first: $first, filters: {productionYears: $productionYears}) {
     payload {
```
Comment (mta-trackunit) [inline guides/sdk-docs/public-apis/graphql-api.md:54]: do we have payload here??

**Review by mta-trackunit — APPROVED** [review]

File: guides/sdk-docs/public-apis/graphql-api.md [hunk 2]
```diff
@@ -81,10 +83,77 @@ const { data, loading, error } = useQuery(GetAssetsByProductionYearDocument, {
...
 For more advanced info on the executor you can read up on how to generate code from [GraphQL codegen cli](https://the-guild.dev/graphql/codegen/docs/getting-started/installation)   
 
-## Example: Full React Component (App.tsx)
+# Mutations
+
+The process for mutation is similar. 
+
+## 1. Add mutation to .graphql file 
+
+You should start with adding a mutation to the `demo.graphql` created above.
```
[the remark this answers is not in the document]
Comment (tandrup) [reply guides/sdk-docs/public-apis/graphql-api.md]: Fixed

File: guides/sdk-docs/public-apis/graphql-api.md (hunk 1 above)
Comment (tandrup) [reply guides/sdk-docs/public-apis/graphql-api.md]: Oh yes. That is an old example.

File: guides/sdk-docs/public-apis/graphql-api.md [hunk 3]
```diff
@@ -81,10 +83,77 @@ const { data, loading, error } = useQuery(GetAssetsByProductionYearDocument, {
...
+To call a preview query you will need to include `TU-PREVIEW` header like this:
+
+```typescript
+const { data, loading, error } = useQuery(GetAssetsByProductionYearDocument, {
+  variables: {
+    // Any variables your query requires
+  },
+  context: { 
+    headers: {
+      "TU-PREVIEW": "<FEATURE-CODE-NAME>" // Look up the feature code name in the schema docs.
```
[the remark this answers is not in the document]
Comment (tandrup) [reply guides/sdk-docs/public-apis/graphql-api.md:151]: Done

**Pull request merged**